src/Contracts/Models/TaskObject.cs

```diff
@@ -1,31 +1,35 @@
-﻿using Newtonsoft.Json;
+﻿// SPDX-FileCopyrightText: © 2021-2022 MONAI Consortium
+// SPDX-License-Identifier: Apache License 2.0
+
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
 
 namespace Monai.Deploy.WorkflowManager.Contracts.Models
 {
     public class TaskObject
     {
-        [JsonProperty(PropertyName = "id")]
+        [ConfigurationKeyName("id")]
```

Comment:
May want to use `JsonProperty` for models instead of `ConfigurationKeyName`

Reply:
Changed